### DISCUSSION TITLE
[Q2 Vite]: vueCompiler: true, does not seem to enable vue compiler

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#13862] [CLI - PWA mode] [answered]
in my quasar.conf.js file, I have set `build: { vueCompiler: true }` but I am still getting this warning when running my dev server:
```
[Vue warn]: The `compilerOptions` config option is only respected when using a build of Vue.js that includes the runtime compiler (aka "full build"). Since you are using the runtime-only build, `compilerOptions` must be passed to `@vue/compiler-dom` in the build setup instead.
```
(this is part of my so far futile attempt to get `app.config.compilerOptions.isCustomElement` to work btw)

**Comment by yusufkandemir** [comment, marked as answer]
> Since you are using the runtime-only build, `compilerOptions` must be passed to `@vue/compiler-dom` in the build setup instead.

You don't need the runtime compiler for this, just pass the build-time options as it says. See [Vue docs](https://vuejs.org/guide/extras/web-components.html#example-vite-config).

https://quasar.dev/quasar-cli-vite/quasar-config-js#build
```js
build: {
  viteVuePluginOptions: {
    template: {
      compilerOptions: {
        isCustomElement: (tag) => tag.startsWith('my-')
      }
    }
  }
}
```

**Reply by yusufkandemir** [reply]
Also, there is no option called `build: { vueCompiler: true }` for app-vite.

**Reply by ssuess** [reply]
THANK YOU!!! I have been struggling to find the correct configuration magic for this for hours and hours.
